packages/cxl-ui/src/components/cxl-marketing-nav.js

```diff
@@ -138,14 +138,17 @@ export class CXLMarketingNavElement extends LitElement {
      */
     const menuItemSearchContextMenu = this.menuItemSearchElement.querySelector(
       'vaadin-context-menu'
-    );
+    ); /* , { once: true } necessary for `content-changed`? */
+
+    /* Mate : focusTrap seems to work ok. Listbox and vaadin-context-menu-item get focus. If we put the listener onto
+    vaadin-context-menu-item and focus input it seems to work. */
 
     menuItemSearchContextMenu.addEventListener(
       'opened-changed',
       ee => {
         const searchForm = ee.target.$.overlay.querySelector('.search-form');
-
-        searchForm.addEventListener('keydown', ef => {
+        searchForm.parentElement.addEventListener('keydown', ef => {
+          searchForm.querySelector('input').focus();
```

Comment:
GH seems to be indicating incorrect spaces -> tabs conversion, aka whitespace errors.

What editor are you using? Must be configured to respect repository `.editorconfig`.

Reply:
Using visual studio code. Installed editorconfig plugin now, hopefully it will work.

Reply:
We are all on IntelliJ PhpStorm - if possible, go with that. Otherwise your tooling support is all on you and will always be worse than the team's.

Reply:
ok,  I have it too, will start using.